Composer/packages/client/__tests__/components/CreationFlow/CreateOptions/index.test.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import { render, fireEvent } from '@bfc/test-utils';
+
+import { StoreContext } from '../../../../src/store';
+import { CreateOptions } from '../../../../src/components/CreationFlow/CreateOptions';
+
+describe('<CreateOptions/>', () => {
+  const handleDismissMock = jest.fn();
+  const handleCreateNextMock = jest.fn();
+  const saveTemplateMock = jest.fn();
+  const templates = [
+    {
+      description: 'empty bot',
+      id: 'EmptyBot',
+      index: 0,
+      name: 'Empty Bot',
+      path: 'templatePath',
+      support: ['C#', 'JS'],
+      tags: ['Basic'],
+    },
+    {
+      description: 'echo bot',
+      id: 'EchoBot',
+      index: 1,
+      name: 'Echo Bot',
+      path: 'templatePath',
+      support: ['C#', 'JS'],
+      tags: ['Basic'],
+    },
+  ];
+  let component, storeContext;
+
+  function renderComponent() {
+    return render(
+      <StoreContext.Provider value={storeContext}>
+        <CreateOptions
+          path="create"
+          templates={templates}
+          onDismiss={handleDismissMock}
+          onNext={handleCreateNextMock}
+        />
+      </StoreContext.Provider>
+    );
+  }
+
+  beforeEach(() => {
+    storeContext = {
+      actions: {
+        saveTemplateId: saveTemplateMock,
+      },
+      state: {
+        templateId: '',
+      },
+    };
+  });
+
+  it('should save empty bot template id', async () => {
+    component = renderComponent();
```

Comment:
Since component is meant to be isolated in each test, make this a `const` instead of assigning.